app/grandchallenge/publications/templates/publications/publication_list.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% load static %}
+{% load publication_extras %}
+{% load guardian_tags %}
+{% load url %}
+{% load bleach %}
+
+
+{% block title %}Publications - {{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item active" aria-current="page">Publications</li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+        {% include "grandchallenge/partials/filters.html" with filter=filter filters_applied=filters_applied %}
+    <div>
+        <h2>Publications</h2>
+    </div>
+    <div class="d-flex justify-content-between my-3">
+        <h6 class="ml-1 mt-2">{{ num_publications }} publications | {{ num_citations }} citations </h6>
+        <a class="btn btn-primary" href="{% url 'publications:create' %}" role="button">Add a publication</a>
+    </div>
+        <ul class="list-group">
+            {% for publication in object_list %}
+                <li class="list-group-item p-0">
+                    <div class="py-2 pl-3">
+                        {{ publication.citation|clean }}
+                    </div>
+                    <div class="pt-1 pb-2 pl-3">
+                        <a class="btn btn-outline-primary btn-sm mb-1 mr-1" href="{{ publication.url }}"><i class="fa fa-link"></i></a>
+                        {% with publication|get_associated_objects as publication_objects %}
+                            {% if publication_objects %}
+                                {% for obj, obj_model in publication_objects.items %}
+                                    {% get_obj_perms request.user for obj as "object_perms" %}

```

Comment:
This is going to hit the database twice for each publication, for each related object, regardless of whether the user has the permission. You can check by adding two publications, then adding those two publications to one object and viewing with a non-superuser (who skips all permissions checks). 

![image](https://user-images.githubusercontent.com/12661555/125608241-8e5ac515-712e-401a-ba61-14b474db4363.png)